db/migrate/20151216095054_create_container_builds.rb

```diff
@@ -0,0 +1,22 @@
+class CreateContainerBuilds < ActiveRecord::Migration
+  def change
+    create_table :container_builds do |t|
+      t.string     :ems_ref
+      t.string     :name
+      t.timestamp  :creation_timestamp
+      t.string     :resource_version
+      t.string     :namespace
+      t.string     :service_account
+      t.string     :build_source_type
+      t.string     :binary
+      t.string     :dockerfile
+      t.string     :git
```

Comment:
@enoodle I prefer the naming to be scoped, for example instead of `git` it should probably be `source_git`, same for the others (`context_dir`, etc.).